File: @LV1/1.8/anewalphabet/anewalphabet.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strings"
)

var dict = map[rune]string{
	'a': "@",
	'b': "8",
	'c': "(",
	'd': "|)",
	'e': "3",
	'f': "#",
	'g': "6",
	'h': "[-]",
	'i': "|",
	'j': "_|",
	'k': "|<",
	'l': "1",
	'm': "[]\\/[]",
	'n': "[]\\[]",
	'o': "0",
	'p': "|D",
	'q': "(,)",
	'r': "|Z",
	's': "$",
	't': "']['",
	'u': "|_|",
	'v': "\\/",
	'w': "\\/\\/",
	'x': "}{",
	'y': "`/",
	'z': "2",
}

func transform(s string) {
	for _, c := range s {
		if dict[c] != "" {
			fmt.Print(dict[c])
		} else {
			fmt.Print(string(c))
		}
	}
	fmt.Println()
}

func main() {
	scanner := bufio.NewScanner(os.Stdin)
	for scanner.Scan() {
		s := scanner.Text()
		transform(strings.ToLower(s))
	}
}

File: @LV1/1.8/anewalphabet/anewalphabet.cpp
#include <cctype>
#include <cstdio>
#include <cstring>
#include <map>
#include <string>
using namespace std;

map<char, string> dict;

void init_dict() {
    dict['a'] = "@";
    dict['b'] = "8";
    dict['c'] = "(";
    dict['d'] = "|)";
    dict['e'] = "3";
    dict['f'] = "#";
    dict['g'] = "6";
    dict['h'] = "[-]";
    dict['i'] = "|";
    dict['j'] = "_|";
    dict['k'] = "|<";
    dict['l'] = "1";
    dict['m'] = "[]\\/[]";
    dict['n'] = "[]\\[]";
    dict['o'] = "0";
    dict['p'] = "|D";
    dict['q'] = "(,)";
    dict['r'] = "|Z";
    dict['s'] = "$";
    dict['t'] = "']['";
    dict['u'] = "|_|";
    dict['v'] = "\\/";
    dict['w'] = "\\/\\/";
    dict['x'] = "}{";
    dict['y'] = "`/";
    dict['z'] = "2";
}

void transform(char *s) {
    for (int i = 0; s[i]; i++) {
        char c = tolower(s[i]);

        if (!strcmp(dict[c].c_str(), "")) {
            printf("%c", c);
        } else {
            printf("%s", dict[c].c_str());
        }
    }
}

int main() {
    init_dict();
    char s[1000];
    fgets(s, 1000, stdin);
    transform(s);
}